### DISCUSSION TITLE
A Journey with friends in search of the Golden Soroban

### DISCUSSION THREAD

**Discussion by OliaXLM** [stellar/sorobanathon#156] [General]
Hello, here is my submisson of a long journey with Bee and Roscoe.

Prologue
https://litemint.com/items/GB4BMZPJG6BM5CGIA5JVHFWNTIVU3JTNT43FNWR53FGIJ7AONCJJG7VT/PROLOGUE

Quest 1
https://litemint.com/items/GCNDAYVY37AWWFYXDFDY73Y5ONDK3KGVDNZQBAUTCPO6YPQWTNEEKG7R/CHAPTER1

Quest 2
https://litemint.com/items/GBACR7SAMYB6A6W3UYC6UEYW323UIMFRPU5U7UYT36DAOXT75W7DLSLE/CHAPTER2

Quest 3
https://litemint.com/items/GDC5X3S6FUSO3ZNMKIUOXPFF6ZJY3R4UFIAX3KEGXONAVPP5YMFWTOCN/CHAPTER3

Quest 4
https://litemint.com/items/GA6MURRRGRW3JKKNQGBSWQUNMZ5M2ZUMOEATICZJCJOFMZKKAHF7WMU3/CHAPTER4

Quest 5
https://litemint.com/items/GBKNMSI25253YGWDORD3WCHMAYCGKGKQL6Z337OTWKJW33FYOZJJKDKB/CHAPTER5

Quest 6
https://litemint.com/items/GAI7Q3L77VL3YB6FBLZFGQHZTHGHB77274D2RSPWE2OEMH3WMOETNZJV/CHAPTER6

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by OliaXLM** [reply]
Thank you!
